guardrails/llm_providers.py

```diff
@@ -831,6 +831,14 @@ async def invoke_llm(
             *args,
             **kwargs,
         )
+        if kwargs.get("stream", False):
+            # If stream is defined and set to True,
+            # the callable returns a generator object
+            # response = cast(AsyncIterable[str], response)
+            return LLMResponse(
+                output="",
+                async_stream_output=response.completion_stream,
+            )
```

Comment:
Will this same logic need to be applied to the other Async Callable classes?

Reply:
Yes, good call. It seems like openai async streaming is not functioning as expected. Put a fixme for now..